package.json

```diff
@@ -1,68 +1,58 @@
 {
   "name": "geoguess",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "GeoGuess is an open-source geography game with Google Map StreetView.",
   "author": "Bilel Jegham and others",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "test:unit": "cross-env NODE_OPTIONS=--unhandled-rejections=warn vue-cli-service test:unit",
+    "serve": "vite serve",
+    "dev": "vite dev",
+    "build": "vite build",
+    "lint": "eslint .",
```

Comment:
Don't see "test:unit" cmd ...

Reply:
vite-jest only works with Vite 2 so this won't work

Reply:
I'll work on migrating to use Vitest